app/bundles/CoreBundle/Assets/css/app/less/components/accordions.less

```diff
@@ -0,0 +1,122 @@
+.accordion {
+    --layout-density-padding-inline-local: 16px;
+    box-sizing: border-box;
```

Comment:
For this component (since it's a completely new addition) I was thinking about ensuring that other changes in base code wouldn't affect the component and would keep it more stable over the time. This means not counting with what the base framework applies

_eg. try to set the display to flex instead of block in any Mautic instance to watch it break and become unusable_

Reply:
If there's a fear of certain base settings (or resets) getting lost when moving past certain Bootstrap versions, it's best to keep these type or global/reset styles separately. And that's a concern for a separate ticket/PR.
It's best practice to use `border-box` and anything else is an exception, so it doesn't make sense to start putting it into components.

Reply:
Makes sense, it's fixed now, thanks!